cmd/crc/cmd/console.go

```diff
@@ -33,21 +33,22 @@ var consoleCmd = &cobra.Command{
 	Short:   "Open the OpenShift Web Console in the default browser",
 	Long:    `Open the OpenShift Web Console in the default browser or print its URL or credentials`,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		return runConsole(os.Stdout, newMachine(), consolePrintURL, consolePrintCredentials, outputFormat)
+		return runConsole(os.Stdout, daemonclient.New(), consolePrintURL, consolePrintCredentials, outputFormat)
 	},
 }
 
-func showConsole(client machine.Client) (*types.ConsoleResult, error) {
-	if err := checkIfMachineMissing(client); err != nil {
+func showConsole(client *daemonclient.Client) (*client.ConsoleResult, error) {
+	if err := checkIfMachineMissingWithDaemon(client); err != nil {
```

Comment:
Instead of adding API to the daemon to check for the machine existence, have you considered moving the VM existence check to the remote `WebconsoleURL` method, and return a `ErrMachineNotExist` from these methods when the VM is not present?

Reply:
Yes, that what I mentioned here https://github.com/crc-org/crc/pull/3392#issuecomment-1291788816

Going to redo this PR